sql/core/src/main/scala/org/apache/spark/sql/catalyst/SQLBuilder.scala

```diff
@@ -193,6 +196,9 @@ class SQLBuilder(logicalPlan: LogicalPlan) extends Logging {
     case OneRowRelation =>
       ""
 
+    case Hint(_, _, child) =>
```

Comment:
After applying the rules of `Analyzer`, there will be no more `Hint` case class.